modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js

```diff
@@ -166,14 +166,26 @@ function Actions({actions, itemData, itemId, menuActive, onMenuActiveChange}) {
 			if (onActionDropdownItemClick) {
 				onActionDropdownItemClick(exposedProps);
 			}
+
+			if (target === 'link' && defaultPrevented) {
+				navigate(url);
+			}
```

Comment:
Without confirmation dialog, this can simply be a link, and behave like default in browser. But, if we do have a confirmation dialog, we need to prevent navigation and activate it only on confirm.

Reply:
Yep, I think there is no case of confirmation + navigation in this type of actions so this change is needed